R/utils-cache.R

```diff
@@ -16,8 +16,9 @@ hash_standardize <- function(text) {
 #' Check if text is cached
 #'
 #' This boils down to check if the hash exists at the caching dir as a file.
-#' @param text,transformers,more_specs Passed to [cache_make_key()] to generate
-#'   a key.
+#' @param text Passed to [cache_make_key()] to generate a key.
+#' @param transformers Passed to [cache_make_key()] to generate a key.
+#' @param more_specs Passed to [cache_make_key()] to generate a key.
```

Comment:
How is this better than documenting each individually?

Reply:
It's not that this is better. But this is no longer supported in the recent version of roxygen2. If you inherit these docs in another function, only the first parameter will be inherited; not the rest.

Reply:
e.g. see [this](https://github.com/r-lib/roxygen2/issues/1347) issue.